xwayland/selection/selection.c

```diff
@@ -33,6 +33,13 @@ void xwm_selection_transfer_destroy_property_reply(
 	transfer->property_reply = NULL;
 }
 
+void xwm_selection_transfer_finished(
```

Comment:
Maybe rename to `xwm_selection_transfer_finish` for consistency with the rest of wlroots?

Reply:
Changed to `_finish`.